env.example

```diff
@@ -223,3 +223,7 @@ JIBRI_XMPP_PASSWORD=
 
 # Jitsi image version (useful for local development)
 #JITSI_IMAGE_VERSION=latest
+
+# Web server's STUN servers to give to the clients, comma-separated list.
+# Used to define config.p2p.stunServers for explicitly specifying STUN servers.
```

Comment:
Drop this comment. This should go in the handbook.

Reply:
Done